### ISSUE TITLE
"Series Watchlist" shows bottom-right time zone wrong

### ISSUE THREAD

**Issue by RoundSparrow** [tarzasai/Uoccin#5]
I'm in USA Central - and when I view the show "Halt and Catch Fire" (and others) it shows "AMC: Sun 04:00 PM".  The show is not until 10:00 PM.

**Comment by tarzasai** [comment]
Yes, in the beginning I thought TVDB was providing airtimes in GMT, but it's not true, sooo... for now the app translate them to local time but I think I should let the user to input what he want.
